### PR TITLE
Adding schemas documentation and example

### LINKED ISSUE #876: Theming topic should be updated
Based on the new theme changes, [this topic](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes/theming-demo.html) should be updated:

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #876

### CODE REVIEW

File: en/components/themes/examples.md [hunk 1]
```diff
@@ -211,12 +211,94 @@ And the result is:
...
+                igx-color:( "secondary", 500)
+            )
+        )
+);
+```
+The component schemas are applied through the global schemas, in order to use the custom one you have to extend one of the globals (`$dark-schema` or `$light-schema`):
+```scss
+    $my-black-schema: extend( $dark-schema, (
+            ...
+            igx-grid: $_black-grid-schema,
```
Comment (simeonoff) [inline en/components/themes/examples.md]: Can you also show where how you define the palette, it's quite important, especially with dark schemas.

Comment (HristoP96) [reply en/components/themes/examples.md]: The palette definition within the schemas section is added

File: en/components/themes/examples.md [hunk 2]
```diff
@@ -277,7 +277,16 @@ The component schemas are applied through the global schemas, in order to use th
...
     );
 ```
-Finally you have to define the `$palette` and set the `$schema` property of the components themes with the new global schema, defined above.
+Finally, you have to define the `$palette`:
+```scss
+...
+$black-color: #000000;
+$aqua-color: #00BFFF;
+
+$black-theme-palette: igx-palette($primary: $black-color, $secondary: $aqua-color);
```
Comment (simeonoff) [inline en/components/themes/examples.md]: Make sure you set the $grays property to #fff. This is important when creating palettes that are gonna be used with dark schemas.

Comment (HristoP96) [reply en/components/themes/examples.md]: Added $grays parameter with and a note for the readers to apply this when they extend the $dark-schema

**Pull request merged**